formatting.cabal

```diff
@@ -48,6 +48,11 @@ common deps
     ghc-options:       -Wno-prepositive-qualified-module
                        -Wno-missing-safe-haskell-mode
 
+flag avoid-double-conversion
```

Comment:
Minor point, but I think `no-double-conversion` or `disable-double-conversion` would be a better flag name